observability/logging/logging-6.1/6x-cluster-logging-deploying-6.1.adoc

```diff
@@ -0,0 +1,60 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="installing-logging-6-1"]
+= Installing Logging
+:context: installing-logging-6-1
+
+toc::[]
+
+{product-title} Operators use custom resources (CRs) to manage applications and their components. You provide high-level configuration and settings through the CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the logic of the Operator. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs to generate CRs.
+
+
+To get started with {logging}, you must install the following Operators:
+
+* {loki-op} to manage your log store.
+* {clo} to manage log collection and forwarding.
+* {coo-first} to manage visualization.
+
+You can use either the {product-title} web console or the {product-title} CLI to install or configure {logging}.
+
+[IMPORTANT]
+====
+You must configure the {clo} after the {loki-op}.
+====
+
+
+ifdef::openshift-origin[]
+[id="prerequisites_cluster-logging-deploying_{context}"]
+== Prerequisites
+* You have downloaded the {cluster-manager-url-pull} as shown in "Obtaining the installation program" in the installation documentation for your platform.
++
+If you have the pull secret, add the `redhat-operators` catalog to the `OperatorHub` custom resource (CR) as shown in "Configuring {product-title} to use Red{nbsp}Hat Operators".
```

Comment:
In the OKD docs, we are moving to the assumption that the user doesn't have a RH pull secret.  See https://github.com/openshift/openshift-docs/pull/89109/files